Dockerfile

```
@@ -1,19 +1,17 @@
 # Development
-FROM golang:1.11.4-alpine AS development
-
-WORKDIR /go/src/github.com/tidepool-org/shoreline
-
-COPY . .
+FROM golang:1.12.7-alpine AS development
 
 RUN apk --no-cache update && \
     apk --no-cache upgrade && \
     apk add build-base git
 
-RUN go get -u github.com/golang/dep/cmd/dep
+# Using Go module (go 1.12 need this variable to be set to enable modules)
+ENV GO111MODULE on
 
-RUN dos2unix build.sh && \
-    dos2unix test.sh && \
-    ./build.sh
+WORKDIR /go/src/github.com/mdblp/shoreline
```

Retested locally the WORKDIR has no influence on the build (we shoreline sources locally and go-common is retrieved from mdblp packages defined in go.mod file) 
```suggestion
WORKDIR /go/src/github.com/tidepool-org/shoreline
```